contrib/awslambda/python/tests/python/pants_test/contrib/awslambda/python/test_python_awslambda_integration.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
+
+from pants.util.contextutil import temporary_dir
+from pants.util.process_handler import subprocess
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+class PythonAWSLambdaIntegrationTest(PantsRunIntegrationTest):
+  @classmethod
+  def hermetic(cls):
+    return True
+
+  def test_awslambda_bundle(self):
+    with temporary_dir() as distdir:
+      config = {
+        'GLOBAL': {
+          'pants_distdir': distdir,
+          'pythonpath': ['%(buildroot)s/contrib/awslambda/python/src/python'],
+          'backend_packages': ['pants.backend.python', 'pants.contrib.awslambda.python'],
+        }
+      }
+
+      command = [
+        'bundle',
+        'contrib/awslambda/python/src/python/pants/contrib/awslambda/python/examples:hello-lambda',
+      ]
+      pants_run = self.run_pants(command=command, config=config)
+      self.assert_success(pants_run)
+      awslambda = os.path.join(distdir, 'hello-lambda.pex')
+      output = subprocess.check_output(env={'PEX_INTERPRETER': '1'}, args=[
```

Comment:
Is the goal here to check the pex is lambdex'ed rather than a normal pex? If so, would you kindly a comment?

Reply:
Will add an appropriate comment.